### DISCUSSION TITLE
Scripts might create DBs with the wrong version

### DISCUSSION THREAD

**Discussion by manuelma** [spine-tools/SpineOpt.jl#877] [Ideas]
It is possible that a script that writes data to a DB uses the wrong version of the SpineOpt data structure. In that case it would be impossible to update the DB automatically.

We need to find a solution that might involve the final user taking some action. If that's the case, we need to find a nice strategy to communicate such actions to them.

**Comment by jkiviluo** [comment]
Should the API function where you open the database connection require a version check? This is probably not clean at the moment, since we don't have a place for version of a 'specific' database (we have a place to express the 'generic' Spine Toolbox data store version). Should we have one? That place should also then say for which tool the database is meant.
